Commit ID: 5cad3b6bead0cbd76ae7be8d920509400eebed41
Change ID: lkzxoutwyxvwzonqvlktnymznzxtmvwn
Author   : Andreas Källberg <anka.213@gmail.com> (2025-08-18 21:36:18)
Committer: Andreas Källberg <anka.213@gmail.com> (2025-08-18 21:38:29)

    Lower bounds for higher unix deps

diff --git a/gf.cabal b/gf.cabal
index cf7c9a755f..0d5df62775 100644
--- a/gf.cabal
+++ b/gf.cabal
@@ -348,7 +348,7 @@
     build-depends:
       terminfo >=0.4.0 && < 0.5
 
-    if impl(ghc >= 9.6.6)
+    if impl(ghc >= 9.6)
       build-depends: unix >= 2.8 && < 2.9
 
     else
